app/keygen/cmd/decrypt.go

```diff
@@ -99,24 +100,24 @@ func printDecryptedKeys() {
 
 	algorithm := ss.Algorithm
 
-	fmt.Println("Algorithm:", algorithm)
-	fmt.Println("---")
+	println("Algorithm:", algorithm)
+	println("---")
```

Comment:
using the simpler `println()` equivalent instead of `fmt.Println()` — I see this as go inspiring from Rust, and I like it.